clr-k8s-examples/README.md

```diff
@@ -14,7 +14,7 @@ Follow instructions in the [Vagrant docs](https://www.vagrantup.com/intro/gettin
 Or, follow our [detailed steps](vagrant.md)
 
 Now you have a 3 node cluster up and running. Each of them have 2 vCPU, 4GB Memory, 2x10GB disks, 1 additional private network.
```

Comment:
@knangia README should be updated i think?

Reply:
yes, will do now. thank you!